src/test/regress/sql/pg16.sql

```diff
@@ -335,3 +335,34 @@ SET search_path TO pg16;
 \set VERBOSITY terse
 SET client_min_messages TO ERROR;
 DROP SCHEMA pg16 CASCADE;
+
```

Comment:
You can add these tests before the lines
```SQL
\set VERBOSITY terse
SET client_min_messages TO ERROR;
DROP SCHEMA pg16 CASCADE;
```

Also, this test will fail because you haven't added the expected output to pg16.out yet.